### PR TITLE
[MIG] 12.0 pos_loyalty

### PR DESCRIPTION
**Pull request by PierrickBrun**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash together adjacent "OCA Transbot.." commits using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

File: pos_loyalty/README.rst [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/184/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows you to define a loyalty program in the point of sale,
+where the customers earn loyalty points and get rewards.
+
+This module is a forward-port to v10 of the pos_loyalty module from Odoo's
```
Comment (pedrobaeza) [inline pos_loyalty/README.rst:31]: Remove here the v10 part. Odoo is releasing in community for v13 this module, so maybe we should wait and backport it.

Comment (PierrickBrun) [reply pos_loyalty/README.rst:31]: Indeed, I think it could be wise to not merge this PR in order to let the module name pos_loyalty free

**Review by fmdl — APPROVED** [review]
fonctional test : ok

File: pos_loyalty/models/loyalty_program.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Copyright 2004-2010 OpenERP SA
+# Copyright 2017 RGB Consulting S.L. (https://www.rgbconsulting.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class LoyaltyProgram(models.Model):
+    _name = 'loyalty.program'
```
Comment (fmdl) [inline pos_loyalty/models/loyalty_program.py:9]: add `_description = 'Loyalty Program'`

File: pos_loyalty/models/loyalty_reward.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+# Copyright 2018 Lambda IS DOOEL <https://www.lambda-is.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models, api, _
+from odoo.exceptions import ValidationError
+
+
+class LoyaltyReward(models.Model):
+    _name = 'loyalty.reward'
+
```
Comment (fmdl) [inline pos_loyalty/models/loyalty_reward.py:12]: `_description = 'Loyalty Reward'`

File: pos_loyalty/models/loyalty_rule.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Copyright 2004-2010 OpenERP SA
+# Copyright 2017 RGB Consulting S.L. (https://www.rgbconsulting.com)
+# Copyright 2018 Lambda IS DOOEL <https://www.lambda-is.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class LoyaltyRule(models.Model):
+    _name = 'loyalty.rule'
```
Comment (fmdl) [inline pos_loyalty/models/loyalty_rule.py:10]: `_description = 'Loyalty Rule'`

**Review by fmdl — CHANGES_REQUESTED** [review]

**Comment by rafaelbn** [comment]
Hi @PierrickBrun , are you going to fix this PR? Thanks ! :smile:

**Comment by PierrickBrun** [comment]
Hi @rafaelbn, I won't work on it since Odoo 13 will be released in approx. 1 month and should include pos_loyalty in CE. It would certainly be better to backport it here then.

**Comment by wpichler** [comment]
> Hi @rafaelbn, I won't work on it since Odoo 13 will be released in approx. 1 month and should include pos_loyalty in CE. It would certainly be better to backport it here then.

I can not find pos_loyalty in Odoo 13 CE - can anyone point me to the source ?

**Comment by wpichler** [comment]
Hi @rafaelbn Why did you closed it ? I can not find pos_loyalty in Odoo 13 CE - so no backport is possible. So this PR should get finished in my opinion

**Comment by rafaelbn** [comment]
Because @PierrickBrun the author of this PR is not going to work in it anymore (he said that). Feel free to open you own PR with this one as origin.

**Pull request closed without merging**